Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -371,19 +374,14 @@ public final class PackageInfoMapper: PackageInfoMapping {
             return nil
         }
 
-        return ProjectDescription.Project(
+        return ProjectDescription.Project.from(
```

Comment:
This is where you should put the logic I have commented in `Project+ManifestMapper.swift`